frontend/src/components/Forms/reports/ReportFilterForm.js

```diff
@@ -0,0 +1,216 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import PropTypes from "prop-types";
+import { Field, reduxForm, formValueSelector } from "redux-form";
+import { Form, Button, Col, Row } from "antd";
+import * as FORM from "@/constants/forms";
+import {
+  getDropdownMineReportCategoryOptions,
+  getMineReportDefinitionOptions,
+} from "@/selectors/staticContentSelectors";
+import { createDropDownList } from "@/utils/helpers";
+import { renderConfig } from "@/components/common/config";
+import CustomPropTypes from "@/customPropTypes";
+
+const propTypes = {
+  handleSubmit: PropTypes.func.isRequired,
+  onSubmit: PropTypes.func.isRequired,
+  reset: PropTypes.func.isRequired,
+  mineReportDefinitionOptions: PropTypes.arrayOf(PropTypes.objectOf(PropTypes.any)).isRequired,
+  dropdownMineReportCategoryOptions: PropTypes.arrayOf(
+    PropTypes.objectOf(CustomPropTypes.dropdownListItem)
+  ).isRequired,
+  selectedMineReportCategory: PropTypes.string.isRequired,
+  selectedMineReportDefinitionGuid: PropTypes.string.isRequired,
+};
+
+const defaultProps = {};
```

Comment:
This can be deleted if you don't have any defaultProps

Reply:
Removed.